src/sentry/llm/vertex.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+import google.auth
+import google.auth.transport.requests
+import requests
+
+from sentry.runner import configure
+
+configure()
+from django.conf import settings
+
+from sentry.llm.base import LLMBase
+
+logger = logging.getLogger(__name__)
+
+
+class VertexProvider(LLMBase):
```

Comment:
can we make sure vertex works well with the AI auto completion use case? you should be able to test it locally on the command line similar to how we did for spam detection. seems like you'd have to abstract out `max_output_tokens` so it can be passed into `complete_prompt`.

Reply:
what url would we use?

Reply:
wdym?

Reply:
tested, and it works!

Reply:
![scr-20240322-lmho_720](https://github.com/getsentry/sentry/assets/56095982/710d1f11-0695-4dca-bdf7-cde2c49cf598)